src/ace-lib/ace.ml

```diff
@@ -283,4 +283,4 @@ let delete doc range =
   doc##(replace range (Js.string ""))
 
 let remove { editor } dir =
-  editor##(remove (Js.string "left"))
+  editor##(remove (Js.string dir))
```

Comment:
Is it the "silent bug"?

Reply:
yes, one of them. The more dangerous one is the closing of file descriptors in the toplevel workers, which actually could close the wrong one.